app/script/auth/page/Verify.js

```diff
@@ -50,7 +50,7 @@ const Verify = ({account, authError, history, isInTeamFlow, intl: {formatMessage
       Promise.resolve()
         .then(() => connected.doRegisterPersonal({...account, email_code}))
         .then(() => {
-          connected.trackEvent({name: TrackingAction.EVENT_NAME.PERSONAL.CREATED});
+          connected.trackEvent({attributes: {context: 'email'}, name: TrackingAction.EVENT_NAME.PERSONAL.CREATED});
```

Comment:
This will have to be adjusted once we bring generic and personal invites back in the coming days.